invenio_rdm_migrator/streams/records/table_generators/records.py

```diff
@@ -214,45 +214,3 @@ def _resolve_communities(communities):
         communities = parent["json"].get("communities")
         if communities:
             _resolve_communities(communities)
-
-    def _cleanup_db(self):
-        """Cleanup DB after load."""
-        # FIXME: abstract to tables, can we do without invenio imports
-        # cant fix atm, versions state needs to be deleted in the middle
-        # from invenio_db import db
-        # from invenio_pidstore.models import PersistentIdentifier
-        # from invenio_rdm_records.records.models import (
-        #     RDMDraftMetadata,
-        #     RDMParentMetadata,
-        #     RDMRecordMetadata,
-        #     RDMVersionsState,
-        # )
-        # PersistentIdentifier.query.filter(
-        #     PersistentIdentifier.pid_type.in_(("recid", "doi", "oai")),
-        #     PersistentIdentifier.object_type == "rec",
-        # ).delete()
-        # RDMVersionsState.query.delete()
-        # RDMRecordMetadata.query.delete()
-        # RDMParentMetadata.query.delete()
-        # RDMDraftMetadata.query.delete()
-        # db.session.commit()
-        pass
-
-    def _cleanup_files(self):
```

Comment:
is this safe to delete? Double checking just in case it was there as a placeholder or something.

Reply:
yes, it is. It was an old idea we had to remove e.g. tmp files or even clean up a db (assumed there would be an intermediate one). So it does not make sense anymore and it was forcing many `pass` implementations.